package.json

```diff
@@ -7,7 +7,10 @@
     "dev": "strapi develop",
     "start": "strapi start",
     "build": "strapi build",
-    "strapi": "strapi"
+    "strapi": "strapi",
+    "types": "npm run types:prod",
+    "types:local": "npx openapi-typescript http://localhost:1337/api/docs/openapi.json -o src/gen/types.ts",
+    "types:prod": "npx openapi-typescript https://cms.cow.fi/api/docs/openapi.json -o src/gen/types.ts"
```

Comment:
I commented in the other PR about this being generated from the running webserver rather than a static file

Reply:
It needs the server to run. This is why I added two, one the production one, the other the local.

I plan to add the steps in a notion to modify the model and regenerate the lib (once all is working)